lunatrace/bsl/backend/src/express-routes/github-routes.ts

```diff
@@ -18,5 +18,5 @@ import { githubLogin } from '../github/login';
 
 export const githubApiRouter = express.Router();
 
-githubApiRouter.get('/github/install', githubInstall);
+githubApiRouter.get('/github/install', express.json(), githubInstall);
```

Comment:
we could just apply this as middleware for all of `githubApiRouter` all requests coming in via github will be json